applications/GeoMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp

```diff
@@ -26,4 +26,19 @@ int ConstitutiveLawUtilities::GetStateVariableIndex(const Variable<double>& rThi
     return index - 1;
 }
 
+void ConstitutiveLawUtilities::SetConstitutiveParameters(ConstitutiveLaw::Parameters& rConstitutiveParameters,
```

Comment:
Very nice addition. I think it could also be used in the `CalculateAnyOfMaterialResponse` functions in the diff order and non-diff order elements!

Reply:
Added there.